armory/datasets/package.py

```diff
@@ -8,6 +8,11 @@
 from armory.logs import log
 from armory.datasets import build, common, upload
 
+# .config and metadata.json are hardcoded values for tfds v4.6.0
+# https://github.com/tensorflow/datasets/blob/v4.6.0/tensorflow_datasets/core/dataset_builder.py#L1257-L1286
```

Comment:
can we clarify in this comment that this is only relevant for tfds datasets that contain builder configs?

Reply:
and maybe rename `DEFAULT_CONFIG_DIR` to `TFDS_...` so it's clear this is a TFDS location we're referring to?

Reply:
check d36fec27